app/views/orders/_cart.html.erb

```diff
@@ -0,0 +1,50 @@
+<% if current_user.nil? %>
+  <% name = "guest" %>
+<% else %>
+  <% name = current_user.name %>
+<% end %>
+
+<%= link_to "Back to products", products_path %>
+
+<h1>Welcome to your cart, <%= name %>! </h1>
+
+<% if @order.order_items.empty? %>
+  You have nothing in your cart, <%= link_to "Would you like to go back to browse products?", products_path %>
+<% else %>
+
+  <% @order.order_items.each do |item| %>
+  <div class="media" id="cart-list">
+    <% product = Product.find(item.product_id) %>
```

Comment:
@roseaboveit Great point! I still forget that you can call the association, and it is super helpful to be reminded!